### ISSUE TITLE
Refused to load the font '<URL>'

### ISSUE THREAD

**Issue by Turecky** [MISP/MISP#7141]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 20.04
| OS version (client)       | Ubuntu 20.04
| PHP version               | 7.4.3
| MISP version / git hash   | v2.4.140 (9f18cec0ca48533eb8e590717cd1e00232a179a6)
| Browser                   | Chrome

### Expected behavior
Font load

### Actual behavior
Refused to load the font '<URL>' because it violates the following Content Security Policy directive: "font-src 'self'".
Refused to load the font 'https://fonts.gstatic.com/s/roboto/v20/KFOiCnqEu92Fr1Mu51QrEz0dL-vwnYh2eg.woff2' because it violates the following Content Security Policy directive: "font-src 'self'".

### Steps to reproduce the behavior
View in Chrome devtool console

### Logs, screenshots, configuration dump, ...
![Screenshot from 2021-03-03 15-36-56](https://user-images.githubusercontent.com/4244630/109813751-69cc3180-7c36-11eb-9134-ced6f04f54e9.png)

**Comment by JakubOnderka** [comment]
Hello, since 2.4.140 there is a new Content-Security-Policy thats block loading any external sources. 

You can allow loading fonts from Google by setting this in `config.php`:

```php
"Security" => [
    "csp" => ["font-src" => "'self' fonts.gstatic.com"],
],
```

**Comment by Turecky** [comment]
Hi!
I added in config as suggested. That take no effect.
error.log still contains lot of records:
`2021-03-03 16:08:40 Error: CSP reported violation from IP 127.10.7.5: {
    "document-uri": "https://localhost/servers/serverSettings/diagnostics",
    "referrer": "https://localhost/servers/serverSettings",
    "violated-directive": "font-src",
    "effective-directive": "font-src",
    "original-policy": "default-src 'self' data: 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; object-src 'none'; frame-ancestors 'none'; worker-src 'none'; child-src 'none'; frame-src 'none'; base-uri 'self'; img-src 'self' data:; font-src 'self'; form-action 'self'; connect-src 'self'; manifest-src 'none'; report-uri /servers/cspReport; upgrade-insecure-requests",
    "disposition": "enforce",
    "blocked-uri": "https://fonts.gstatic.com/s/roboto/v20/KFOiCnqEu92Fr1Mu51QrEz0dL-vwnYh2eg.woff2",
    "status-code": 0,
    "script-sample": ""
}
`

**Comment by JakubOnderka** [comment]
Sorry, there is bug with custom policies. Patch for that #7142 was merged into 2.4 branch.
